library/pkparse.c

```diff
@@ -148,6 +160,8 @@ int mbedtls_pk_parse_public_keyfile( mbedtls_pk_context *ctx, const char *path )
     size_t n;
     unsigned char *buf;
 
+    PK_VALIDATE_RET( ctx != NULL );
```

Comment:
We should check for `path != NULL` here (even if we check it in `mbedtls_pk_load_file()`, too - that's an implementation detail).

Reply:
as above